framework/src/modules/chain/blocks/chain.js

```diff
@@ -192,22 +192,19 @@ const applyConfirmedStep = async (storage, slots, block, exceptions, tx) => {
  * @param {function} tx - Database transaction
```

Comment:
Can you update the params and the description for them? 
The most important one is the `shouldSave` flag